From 6e1aaab0ed325e0d2179ac399238bdaeb2b1eac4 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 11 Oct 2024 16:02:18 +0800
Subject: [PATCH 09/12] AOSCOS: chore: refresh documentation and command-line
 instructions

- Fix up grammar and prose in the man page.
- Removing most unneeded dashes (-) in non-compound nouns.
- Fix casing for proper nouns (kmscon => KMSCON, linux => Linux).
- Update options in accordance with the latest code.
- Correct XKB code for plus (not Plus) and minus (not Minus).
---
 docs/man/kmscon.1.xml.in | 260 +++++++++++++++++++++------------------
 src/kmscon_conf.c        |  24 ++--
 2 files changed, 149 insertions(+), 135 deletions(-)

diff --git a/docs/man/kmscon.1.xml.in b/docs/man/kmscon.1.xml.in
index c6ddb0f..60b09aa 100644
--- a/docs/man/kmscon.1.xml.in
+++ b/docs/man/kmscon.1.xml.in
@@ -7,17 +7,25 @@
   Dedicated to the Public Domain
 -->
 
+<!--
+  Revised 2024 by Mingcong Bai <jeffbai@aosc.io>
+-->
+
 <refentry id="kmscon">
   <refentryinfo>
-    <title>kmscon</title>
-    <productname>kmscon</productname>
-    <date>December 2012</date>
+    <title>KMSCON</title>
+    <productname>KMSCON</productname>
+    <date>October 2024</date>
     <authorgroup>
       <author>
         <contrib>Developer</contrib>
         <firstname>David</firstname>
         <surname>Herrmann</surname>
         <email>dh.herrmann@googlemail.com</email>
+        <contrib>Developer</contrib>
+        <firstname>Mingcong</firstname>
+        <surname>Bai</surname>
+        <email>jeffbai@aosc.io</email>
       </author>
     </authorgroup>
   </refentryinfo>
@@ -43,21 +51,21 @@
 
   <refsect1>
     <title>Description</title>
-    <para>kmscon is a KMS/DRM-based system console with an integrated terminal
-          emulator for linux operating systems. It was designed as a replacement
-          for the linux kernel console and virtual terminals.</para>
+    <para>KMSCON is a KMS/DRM-based system console with an integrated terminal
+          emulator for Linux operating systems. It was designed as a replacement
+          for the Linux kernel console and virtual terminals.</para>
 
-    <para>When run in default mode, kmscon allocates a virtual terminal and
+    <para>When run in default mode, KMSCON allocates a virtual terminal and
           provides a terminal emulator on it. It can thus be used as replacement
-          for the linux console and
+          for the Linux console and
           <citerefentry><refentrytitle>agetty</refentrytitle><manvolnum>1</manvolnum></citerefentry>
           on virtual terminals.
-          Compared to the linux console, kmscon provides a rich set of enhanced
+          Compared to the Linux console, KMSCON provides a rich set of enhanced
           features including full internationalized keyboard support, full UTF-8
           input/font support, hardware-accelerated rendering, multi-seat
           support and more.</para>
 
-    <para>When run in listener mode, kmscon watches the system for new seats and
+    <para>When run in listener mode, KMSCON watches the system for new seats and
           automatically provides a system console on it.</para>
   </refsect1>
 
@@ -74,7 +82,7 @@
         <term><option>-h</option></term>
         <term><option>--help</option></term>
         <listitem>
-          <para>Prints a short help text and exits. (default: off)</para>
+          <para>Print a short help text and exit.</para>
         </listitem>
       </varlistentry>
 
@@ -89,7 +97,7 @@
       <varlistentry>
         <term><option>--debug</option></term>
         <listitem>
-          <para>Print debug messages. Works only if kmscon was compiled with
+          <para>Print debug messages. Works only if KMSCON was compiled with
                 debug mode enabled. (default: off)</para>
         </listitem>
       </varlistentry>
@@ -111,7 +119,7 @@
       <varlistentry>
         <term><option>--listen</option></term>
         <listitem>
-          <para>Run kmscon in listen-mode. See section below for more
+          <para>Run KMSCON in listen-mode. See section below for more
                 information. (default: off)</para>
         </listitem>
       </varlistentry>
@@ -123,16 +131,16 @@
       <varlistentry>
         <term><option>--vt {/path/to/vt}</option></term>
         <listitem>
-          <para>Use this VT. If not specified, kmscon tries to find a VT by
-                itself. On seats without VTs, kmscon simply activates itself
+          <para>Use this VT. If not specified, KMSCON tries to find a VT by
+                itself. On seats without VTs, KMSCON simply activates itself
                 automatically without using any VT. Please note that you cannot
-                use this option if kmscon runs on multiple seats
+                use this option if KMSCON runs on multiple seats
                 (see <option>--seats</option>). To avoid this, specify this
                 option in a seat configuration
                 <filename>@CONFIG_DIR@/{seat}.kmscon.conf</filename>.</para>
           <para>You must not run multiple applications on a single VT,
                 otherwise, you might get an unresponsive system. Also note, by
-                default, kmscon tries to use it's controlling terminal as VT and
+                default, KMSCON tries to use it's controlling terminal as VT and
                 if that fails, it tries to allocate a new unused VT.</para>
         </listitem>
       </varlistentry>
@@ -140,9 +148,9 @@
       <varlistentry>
         <term><option>--switchvt</option></term>
         <listitem>
-          <para>If kmscon is started on a VT that is currently not active, this
-                option makes kmscon activate this VT during startup. The same
-                way, during shutdown kmscon will reactivate the VT that was
+          <para>If KMSCON is started on a VT that is currently not active, this
+                option makes KMSCON activate this VT during startup. The same
+                way, during shutdown KMSCON will reactivate the VT that was
                 previously active. (default: on)</para>
         </listitem>
       </varlistentry>
@@ -150,14 +158,14 @@
       <varlistentry>
         <term><option>--seats {list,of,seats}</option></term>
         <listitem>
-          <para>List of seats kmscon will run on. Use 'all' to make kmscon run
-                on all seats. Use 'all,' to make kmscon run on a seat called
-                'all' (empty entries are ignored). Use 'current' to make kmscon
-                run on the current seat. (default: "current")</para>
+          <para>List of seats KMSCON will run on. Use `all' to make KMSCON run
+                on all seats. Use `all,' to make KMSCON run on a seat called
+                `all' (empty entries are ignored). Use `current' to make kmscon
+                run on the current seat. (default: current)</para>
 
-          <para>If kmscon runs on multiple seats, all seats are independent of
-                each other. You can either run a different kmscon process on
-                each seat or make a single kmscon process run on multiple seats.
+          <para>If KMSCON runs on multiple seats, all seats are independent of
+                each other. You can either run a different KMSCON process on
+                each seat or make a single KMSCON process run on multiple seats.
                 In the latter case, global state can be shared to reduce memory
                 consumption (like glyph-caches).</para>
         </listitem>
@@ -222,7 +230,7 @@
         <term><option>-t {term}</option></term>
         <term><option>--term {term}</option></term>
         <listitem>
-          <para>This changes the <varname>$TERM</varname> environment variable
+          <para>Specify the <varname>$TERM</varname> environment variable
                 that is set on new terminal sessions.
                 (default: xterm-256color)</para>
         </listitem>
@@ -239,7 +247,7 @@
       <varlistentry>
         <term><option>--sb-size {size}</option></term>
         <listitem>
-          <para>Maximum scrollback-buffer line count. (default: 1000)</para>
+          <para>Specify the size of the scrollback buffer (in lines). (default: 1000)</para>
         </listitem>
       </varlistentry>
     </variablelist>
@@ -250,7 +258,7 @@
       <varlistentry>
         <term><option>--xkb-model {model}</option></term>
         <listitem>
-          <para>Keyboard model. This should normally not be changed.
+          <para>Specify the keyboard (xkb) model. This should normally not be changed.
                 (default: {xkb default})</para>
         </listitem>
       </varlistentry>
@@ -258,7 +266,7 @@
       <varlistentry>
         <term><option>--xkb-layout {layout}</option></term>
         <listitem>
-          <para>Keyboard layout. (default: {xkb default})</para>
+          <para>Specify the keyboard (xkb) layout. (default: {xkb default})</para>
           <para>German keyboard example: --xkb-layout=de</para>
         </listitem>
       </varlistentry>
@@ -266,14 +274,14 @@
       <varlistentry>
         <term><option>--xkb-variant {variant}</option></term>
         <listitem>
-          <para>Keyboard variant. (default: {xkb default})</para>
+          <para>Speicfy the keyboard (xkb) variant. (default: {xkb default})</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--xkb-options {options}</option></term>
         <listitem>
-          <para>Keyboard options. (default: {xkb default})</para>
+          <para>Specify other keyboard (xkb) options. (default: {xkb default})</para>
         </listitem>
       </varlistentry>
 
@@ -314,24 +322,24 @@
       <varlistentry>
         <term><option>--xkb-repeat-rate {rate}</option></term>
         <listitem>
-          <para>Time between two key-repeats (in milliseconds).
+          <para>Time between two key repeats (in milliseconds).
                 (default: 50)</para>
         </listitem>
       </varlistentry>
     </variablelist>
 
-    <para>Grabs / Keyboard-Shortcuts:</para>
-    <para>All keyboard shortcuts (here: grabs) in kmscon can be modified with
+    <para>Grabs / Keyboard Shortcuts:</para>
+    <para>All keyboard shortcuts (here: grabs) in KMSCON can be modified with
           these options. The argument to such an option is a list of
-          comma-separated
-          keyboard-shortcuts that should trigger the specified action. To
-          disable a shortcut, pass an empty argument (empty list). A keyboard
-          shortcut exists of a list of modifiers followed by a single key. All
+          comma-separated keyboard shortcuts that should trigger the specified
+          action.</para>
+    <para>To disable a shortcut, pass an empty argument (empty list). A keyboard
+          shortcut consists of a list of modifiers followed by a single key. All
           modifiers must be down when the key is pressed for the shortcut to be
           detected. A modifier is surrounded by '&lt;' and '&gt;' (like
           '&lt;shift&gt;'). A key is the name of a keysym (like 'a' or
-          'Return'). Keysyms are case-sensitive and kmscon will suggest a keysym
-          name if you entered a wrong case. However, kmscon can not always guess
+          'Return'). Keysyms are case-sensitive and KMSCON will suggest a keysym
+          name if you entered a wrong case. However, KMSCON can not always guess
           the right case as lower-case-only keysyms are ambiguous.</para>
 
     <para>If a key generates multiple keysyms in your keyboard layout, you can
@@ -339,8 +347,8 @@
           'H+E+L+L+O'). This does not mean that all the listed keys must be
           pressed! It's rather a way to match multi-keysym keys.</para>
 
-    <para>Valid modifiers are: Shift, Control/Ctrl, Alt, Logo (Windows/Apple
-          key)</para>
+    <para>Valid modifiers: Shift, Control/Ctrl, Alt, Logo (Windows/Apple
+          key).</para>
 
     <variablelist>
       <varlistentry>
@@ -379,7 +387,7 @@
         <term><option>--grab-zoom-in {grab}</option></term>
         <listitem>
           <para>Increase font size of the current terminal.
-                (default: &lt;Ctrl&gt;Plus)</para>
+                (default: &lt;Ctrl&gt;plus)</para>
         </listitem>
       </varlistentry>
 
@@ -387,14 +395,14 @@
         <term><option>--grab-zoom-out {grab}</option></term>
         <listitem>
           <para>Decrease font size of the current terminal.
-                (default: &lt;Ctrl&gt;Minus)</para>
+                (default: &lt;Ctrl&gt;minus)</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--grab-session-next {grab}</option></term>
         <listitem>
-          <para>Switch to next session.
+          <para>Switch to the next session.
                 (default: &lt;Ctrl&gt;&lt;Logo&gt;Right)</para>
         </listitem>
       </varlistentry>
@@ -402,7 +410,7 @@
       <varlistentry>
         <term><option>--grab-session-prev {grab}</option></term>
         <listitem>
-          <para>Switch to previous session.
+          <para>Switch to the previous session.
                 (default: &lt;Ctrl&gt;&lt;Logo&gt;Left)</para>
         </listitem>
       </varlistentry>
@@ -410,7 +418,7 @@
       <varlistentry>
         <term><option>--grab-session-dummy {grab}</option></term>
         <listitem>
-          <para>Switch to dummy session (black screen).
+          <para>Switch to a dummy session (blank screen).
                 (default: &lt;Ctrl&gt;&lt;Logo&gt;Escape)</para>
         </listitem>
       </varlistentry>
@@ -418,7 +426,7 @@
       <varlistentry>
         <term><option>--grab-session-close {grab}</option></term>
         <listitem>
-          <para>Close current session (press twice to kill it).
+          <para>Press twice to close the current session.
                 (default: &lt;Ctrl&gt;&lt;Logo&gt;BackSpace)</para>
         </listitem>
       </varlistentry>
@@ -438,8 +446,9 @@
       <varlistentry>
         <term><option>--drm</option></term>
         <listitem>
-          <para>Use DRM devices as display devices, too. fbdev devices are
-                always used, but DRM devices are only used if this option is on.
+          <para>Use DRM (direct rendering manager) devices (usually GPUs) as
+                display devices. fbdev (framebuffer) devices are always used,
+                but DRM devices are only used if this option is enabled.
                 (default: on)</para>
         </listitem>
       </varlistentry>
@@ -447,31 +456,34 @@
       <varlistentry>
         <term><option>--hwaccel</option></term>
         <listitem>
-          <para>Use 3D hardware acceleration. Note that this should be used in
-                conjunction with <option>--drm</option>. On newer hardware, this
-                can speed up rendering by multiple orders of magnitude. However,
-                on older hardware this might even slow down rendering speed.
-                (default: off)</para>
+          <para>Use 3D hardware acceleration. On newer hardware, this can speed
+                up rendering by multiple orders of magnitude. However, on older
+                hardware this might slow down rendering speed. (default: off)</para>
+          <para>Note: This option should be enabled in conjunction with
+                <option>--drm</option></para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--gpus {all,aux,primary}</option></term>
         <listitem>
-          <para>GPU selection algorithm. If 'all' then all found GPUs are used.
-                If 'aux' then primary and auxiliary GPUs are used. If 'primary'
-                then only primary GPUs are used. Primary GPUs are the default
-                GPU on a seat. There is always only one primary GPU on a seat.
-                Auxiliary GPUs are hotpluggable GPUs that provide additional
-                independent displays. (default: all)</para>
+          <para>Specify which GPU(s) could be used for terminal output. If `all'
+                is specified, then all GPUs found are used. If `aux' is
+                specified, then both primary and auxiliary GPUs are used. If
+                `primary' is specified, then only primary GPUs are used.
+                (default: all)</para>
+          <para>Note: Primary GPUs are the default GPU on a seat. There is
+                always only one primary GPU on a seat. Auxiliary GPUs are hot-
+                pluggable GPUs that provide additional independent displays.</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--render-engine {engine}</option></term>
         <listitem>
-          <para>Select console render engine. Available engines are 'bblit',
-                'bbulk' and 'gltex'. (default: detect by GPU type)</para>
+          <para>Specify console the rendering engine. Available engines are
+                `bblit', `bbulk', `gltex', and `pixman'.
+                (default: detect by GPU type)</para>
         </listitem>
       </varlistentry>
 
@@ -490,30 +502,32 @@
       <varlistentry>
         <term><option>--font-engine {engine}</option></term>
         <listitem>
-          <para>Select font-engine. Available engines are 'pango',
-                'unifont' and '8x16'. (default: pango)</para>
+          <para>Specify the font rendering engine. Available engines are
+                `pango', `unifont', and `8x16'. (default: pango)</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--font-size {points}</option></term>
         <listitem>
-          <para>Font size in points. (default: 12)</para>
+          <para>Font size in points (pt). (default: 12)</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--font-name {name}</option></term>
         <listitem>
-          <para>Font name. (default: monospace)</para>
+          <para>Name of the font to be used, note that Fontconfig fallback may
+                be used if the `pango' font rendering engine is specified.
+                (default: monospace)</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--font-dpi {dpi}</option></term>
         <listitem>
-          <para>DPI used for font-rendering. Per monitor DPI values overwrite
-                this global default. (default: 96)</para>
+          <para>DPI (dots-per-inch) used for font-rendering. Per monitor DPI
+                values override this global default. (default: 96)</para>
         </listitem>
       </varlistentry>
     </variablelist>
@@ -524,20 +538,20 @@
       <varlistentry>
         <term><option>--palette {name}</option></term>
         <listitem>
-          <para>Change the default color palette. Setting this to
-                <literal>custom</literal> takes the color palette from the
-                following options.</para>
+          <para>Specify the default color palette. Specify
+                <literal>custom</literal> to make use of the color palette from
+                the following options.</para>
         </listitem>
       </varlistentry>
 
-      <para>In the following options <replaceable>color</replaceable> is an RGB
-            color value written as three comma-separated integers between 0 and
-            255 inclusive.</para>
+      <para>In the following options, <replaceable>colors</replaceable> are
+            specified in RGB color value, written in three comma-separated
+            integers between 0 and 255 inclusive.</para>
 
       <varlistentry>
         <term><option>--palette-black {color}</option></term>
         <listitem>
-          <para>Change the black color in the custom palette to
+          <para>Specify the black color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 0, 0)</para>
         </listitem>
@@ -546,7 +560,7 @@
       <varlistentry>
         <term><option>--palette-red {color}</option></term>
         <listitem>
-          <para>Change the red color in the custom palette to
+          <para>Specify the red color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 205, 0, 0)</para>
         </listitem>
@@ -555,7 +569,7 @@
       <varlistentry>
         <term><option>--palette-green {color}</option></term>
         <listitem>
-          <para>Change the green color in the custom palette to
+          <para>Specify the green color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 205, 0)</para>
         </listitem>
@@ -564,7 +578,7 @@
       <varlistentry>
         <term><option>--palette-yellow {color}</option></term>
         <listitem>
-          <para>Change the yellow color in the custom palette to
+          <para>Specify the yellow color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 205, 205, 0)</para>
         </listitem>
@@ -573,7 +587,7 @@
       <varlistentry>
         <term><option>--palette-blue {color}</option></term>
         <listitem>
-          <para>Change the blue color in the custom palette to
+          <para>Specify the blue color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 0, 238)</para>
         </listitem>
@@ -582,7 +596,7 @@
       <varlistentry>
         <term><option>--palette-magenta {color}</option></term>
         <listitem>
-          <para>Change the magenta color in the custom palette to
+          <para>Specify the magenta color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 205, 0, 205)</para>
         </listitem>
@@ -591,7 +605,7 @@
       <varlistentry>
         <term><option>--palette-cyan {color}</option></term>
         <listitem>
-          <para>Change the cyan color in the custom palette to
+          <para>Specify the cyan color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 205, 205)</para>
         </listitem>
@@ -600,16 +614,16 @@
       <varlistentry>
         <term><option>--palette-light-grey {color}</option></term>
         <listitem>
-          <para>Change the light grey color in the custom palette to
+          <para>Specify the light grey color in the custom palette to
                 <replaceable>color</replaceable>.
-                (default: 229, 229,229)</para>
+                (default: 229, 229, 229)</para>
         </listitem>
       </varlistentry>
 
       <varlistentry>
         <term><option>--palette-dark-grey {color}</option></term>
         <listitem>
-          <para>Change the dark grey color in the custom palette to
+          <para>Specify the dark grey color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 127, 127, 127)</para>
         </listitem>
@@ -618,7 +632,7 @@
       <varlistentry>
         <term><option>--palette-light-red {color}</option></term>
         <listitem>
-          <para>Change the light red color in the custom palette to
+          <para>Specify the light red color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 255, 0, 0)</para>
         </listitem>
@@ -627,7 +641,7 @@
       <varlistentry>
         <term><option>--palette-light-green {color}</option></term>
         <listitem>
-          <para>Change the light green color in the custom palette to
+          <para>Specify the light green color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 255, 0)</para>
         </listitem>
@@ -636,7 +650,7 @@
       <varlistentry>
         <term><option>--palette-light-yellow {color}</option></term>
         <listitem>
-          <para>Change the light yellow color in the custom palette to
+          <para>Specify the light yellow color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 255, 255, 0)</para>
         </listitem>
@@ -645,7 +659,7 @@
       <varlistentry>
         <term><option>--palette-light-blue {color}</option></term>
         <listitem>
-          <para>Change the light blue color in the custom palette to
+          <para>Specify the light blue color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 92, 92, 255)</para>
         </listitem>
@@ -654,7 +668,7 @@
       <varlistentry>
         <term><option>--palette-light-magenta {color}</option></term>
         <listitem>
-          <para>Change the light magenta color in the custom palette to
+          <para>Specify the light magenta color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 255, 0, 255)</para>
         </listitem>
@@ -663,7 +677,7 @@
       <varlistentry>
         <term><option>--palette-light-cyan {color}</option></term>
         <listitem>
-          <para>Change the light cyan color in the custom palette to
+          <para>Specify the light cyan color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 0, 255, 255)</para>
         </listitem>
@@ -672,7 +686,7 @@
       <varlistentry>
         <term><option>--palette-white {color}</option></term>
         <listitem>
-          <para>Change the white color in the custom palette to
+          <para>Specify the white color in the custom palette to
                 <replaceable>color</replaceable>.
                 (default: 255, 255, 255)</para>
         </listitem>
@@ -681,7 +695,7 @@
       <varlistentry>
         <term><option>--palette-foreground {color}</option></term>
         <listitem>
-          <para>Change the default foreground color in the custom
+          <para>Specify the default foreground color in the custom
                 palette to <replaceable>color</replaceable>.
                 (default: 229, 229, 229)</para>
         </listitem>
@@ -690,7 +704,7 @@
       <varlistentry>
         <term><option>--palette-background {color}</option></term>
         <listitem>
-          <para>Change the default background color in the custom
+          <para>Specify the default background color in the custom
                 palette to <replaceable>color</replaceable>.
                 (default: 0, 0, 0)</para>
         </listitem>
@@ -700,12 +714,12 @@
 
   <refsect1>
     <title>Configuration</title>
-    <para>Default configuration directory is <filename>@CONFIG_DIR@</filename>
-          and the main configuration file is called
+    <para>The default configuration directory is <filename>@CONFIG_DIR@</filename>
+          and the main configuration file is named
           <filename>@CONFIG_DIR@/kmscon.conf</filename>. You can provide an
           additional configuration file for each seat via
           <filename>@CONFIG_DIR@/{seatname}.kmscon.conf</filename>.
-          Any command-line option that can be passed to kmscon can also be put
+          Any command-line option that can be passed to KMSCON can also be put
           into those configuration files.  For example:</para>
 
     <programlisting>
@@ -715,37 +729,37 @@
       palette-foreground=255, 0, 0
     </programlisting>
 
-    <para>Command-line options overwrite
-          configuration file options. And per-seat configuration files overwrite
+    <para>Command-line options override
+          configuration file options. And per-seat configuration files override
           the global configuration file.</para>
   </refsect1>
 
   <refsect1>
     <title>Default Mode</title>
-    <para>In default mode, kmscon acts as a replacement for
+    <para>In default mode, KMSCON acts as a replacement for
           <citerefentry><refentrytitle>agetty</refentrytitle><manvolnum>1</manvolnum></citerefentry>.
-          kmscon opens a virtual terminal and
+          KMSCON opens a virtual terminal and
           provides a system console on it. It replaces the linux-console on this
           VT. You can select the virtual terminal
-          via the <option>--vt</option> option. kmscon will seamlessly integrate
+          via the <option>--vt</option> option. KMSCON will seamlessly integrate
           into existing setups so you can use X, Wayland or any other Graphics
           Systems simultaneously. You can even use the linux-console on all the
           other virtual terminals.</para>
 
-    <para>By default, kmscon runs on
+    <para>By default, KMSCON runs on
           <emphasis>seat0</emphasis> but you can change the seat via the
           <option>--seats</option> option. For seats without virtual terminals,
-          kmscon will automatically activate itself and run exclusively on that
+          KMSCON will automatically activate itself and run exclusively on that
           seat. If the virtual terminal or the seat that kmscon
-          runs on is gone, kmscon will automatically exit. See the
+          runs on is gone, KMSCON will automatically exit. See the
           <emphasis>Listener-Mode</emphasis> to avoid this.</para>
 
-    <para>kmscon spawns
+    <para>KMSCON spawns
           <citerefentry><refentrytitle>login</refentrytitle><manvolnum>1</manvolnum></citerefentry>
-          on each new terminal. After a session exits, kmscon automatically
+          on each new terminal. After a session exits, KMSCON automatically
           respawns the next session. You can change the program that is spawned
           with the <option>--login</option> option. To stop kmscon, send SIGTERM
-          to the kmscon process.</para>
+          to the KMSCON process.</para>
 
     <para>Keyboard input is done by <emphasis>libxkbcommon</emphasis>. This
           provides the same features as the X11 Keyboard System but without any
@@ -755,32 +769,32 @@
           <option>--xkb-layout=de</option> changes the keyboard layout to
           German.</para>
 
-    <para>kmscon uses KMS/DRM to access graphics devices (like the X-Server
-          does). Compared to the linux console this allows kmscon to provide
+    <para>KMSCON uses KMS/DRM to access graphics devices (like the X-Server
+          does). Compared to the Linux console this allows KMSCON to provide
           much better font-rendering and enhanced monitor control. However, for
-          backwards compatibility, kmscon can also use fbdev devices.
+          backwards compatibility, KMSCON can also use fbdev devices.
           Hotpluggable graphics devices are marked as auxiliary devices, primary
           GPUs on a seat are marked as primary. Other GPUs stay unmarked. By
-          default kmscon uses all graphics devices but you can make it use only
+          default KMSCON uses all graphics devices but you can make it use only
           primary or auxiliary devices with the <option>--gpus</option>
           option.</para>
   </refsect1>
 
   <refsect1>
     <title>Listener Mode</title>
-    <para>If started in listen-mode (<option>--listen</option>) kmscon will run
+    <para>If started in listen-mode (<option>--listen</option>), KMSCON will run
           as system daemon and provide a system console on each seat that shows
           up if that seat does not have virtual terminals. On seats with virtual
-          terminals you should run kmscon in default mode.</para>
+          terminals you should run KMSCON in default mode.</para>
 
     <para>Several options are exclusively limited to listen-mode, other than
-          these, kmscon behaves the same way as in default-mode.</para>
+          these, KMSCON behaves the same way as in default-mode.</para>
   </refsect1>
 
   <refsect1>
     <title>Sessions</title>
-    <para>If kmscon is active on a seat, the internal session-manager is woken
-          up. At most times, only the terminal-session is active, but kmscon can
+    <para>If KMSCON is active on a seat, the internal session-manager is woken
+          up. At most times, only the terminal-session is active, but KMSCON can
           also support any other session type. You can switch between sessions
           with keyboard-shortcuts and you can create/destroy sessions during
           runtime.</para>
@@ -796,14 +810,14 @@
 
   <refsect1>
     <title>Virtual Terminals</title>
-    <para>Historically, linux provides virtual terminals (VT) only on seat
+    <para>Historically, Linux provides virtual terminals (VT) only on seat
           <emphasis>seat0</emphasis>. This is because only one VT can be active
           at a time but in multi-seat environments you want multiple active VTs,
-          one per seat. It is easy to run kmscon on seat0 on a VT, however, if
-          you want to run kmscon on another seat, there are no VTs. This means,
+          one per seat. It is easy to run KMSCON on seat0 on a VT, however, if
+          you want to run KMSCON on another seat, there are no VTs. This means,
           you can only run one application that uses graphics devices on these
           seats. In most environments this is the X-Server. But if you want to
-          run kmscon together with the X-Server (or any other graphics server),
+          run KMSCON together with the X-Server (or any other graphics server),
           you need a mechanism to notify these servers when they are active.
           This is done via VT APIs on seat0. But note that if the kernel is
           compiled without CONFIG_VT (which controls whether VTs are available),
diff --git a/src/kmscon_conf.c b/src/kmscon_conf.c
index 325e0ec..0dd0db3 100644
--- a/src/kmscon_conf.c
+++ b/src/kmscon_conf.c
@@ -109,7 +109,7 @@ static void print_help()
 		"\t    --xkb-repeat-delay <msecs> [250]\n"
 		"\t                                 Initial delay for key-repeat in ms\n"
 		"\t    --xkb-repeat-rate <msecs>  [50]\n"
-		"\t                                 Delay between two key-repeats in ms\n"
+		"\t                                 Delay between two key repeats in ms\n"
 		"\n"
 		"Grabs / Keyboard-Shortcuts:\n"
 		"\t    --grab-scroll-up <grab>     [<Shift>Up]\n"
@@ -120,24 +120,24 @@ static void print_help()
 		"\t                                  Shortcut to scroll page up\n"
 		"\t    --grab-page-down <grab>     [<Shift>Next]\n"
 		"\t                                  Shortcut to scroll page down\n"
-		"\t    --grab-zoom-in <grab>       [<Ctrl>Plus]\n"
+		"\t    --grab-zoom-in <grab>       [<Ctrl>plus]\n"
 		"\t                                  Shortcut to increase font size\n"
-		"\t    --grab-zoom-out <grab>      [<Ctrl>Minus]\n"
+		"\t    --grab-zoom-out <grab>      [<Ctrl>minus]\n"
 		"\t                                  Shortcut to decrease font size\n"
 		"\t    --grab-session-next <grab>  [<Ctrl><Logo>Right]\n"
-		"\t                                  Switch to next session\n"
+		"\t                                  Switch to the next session\n"
 		"\t    --grab-session-prev <grab>  [<Ctrl><Logo>Left]\n"
-		"\t                                  Switch to previous session\n"
+		"\t                                  Switch to the previous session\n"
 		"\t    --grab-session-dummy <grab> [<Ctrl><Logo>Escape]\n"
-		"\t                                  Switch to dummy session\n"
+		"\t                                  Switch to a dummy session\n"
 		"\t    --grab-session-close <grab> [<Ctrl><Logo>BackSpace]\n"
 		"\t                                  Close current session\n"
 		"\t    --grab-terminal-new <grab>  [<Ctrl><Logo>Return]\n"
-		"\t                                  Create new terminal session\n"
+		"\t                                  Create a new terminal session\n"
 		"\n"
 		"Video Options:\n"
 		"\t    --drm                   [on]    Use DRM if available\n"
-		"\t    --hwaccel               [off]   Use 3D hardware-acceleration if\n"
+		"\t    --hwaccel               [off]   Use 3D hardware acceleration if\n"
 		"\t                                    available\n"
 		"\t    --gpus={all,aux,primary}[all]   GPU selection mode\n"
 		"\t    --render-engine <eng>   [-]     Console renderer\n"
@@ -145,9 +145,9 @@ static void print_help()
 		"\n"
 		"Font Options:\n"
 		"\t    --font-engine <engine>  [pango]\n"
-		"\t                              Font engine\n"
+		"\t                              Font rendering engine\n"
 		"\t    --font-size <points>    [12]\n"
-		"\t                              Font size in points\n"
+		"\t                              Font size in points (pt)\n"
 		"\t    --font-name <name>      [monospace]\n"
 		"\t                              Font name\n"
 		"\t    --font-dpi <dpi>        [96]\n"
@@ -156,8 +156,8 @@ static void print_help()
 		"Palette Options:\n"
 		"\t    --palette <name>                [default]\n"
 		"\t                                      Select the used color palette.\n"
-		"\t                                      'custom' uses the following color\n"
-		"\t                                      options.\n"
+		"\t                                      specify `custom' to use the \n"
+		"\t                                      following color options\n"
 		"\t    --palette-black <color>         [  0,   0,   0]\n"
 		"\t                                      Black in custom palette\n"
 		"\t    --palette-red <color>           [205,   0,   0]\n"
-- 
2.47.0

